src/components/EarnReport.tsx

```diff
@@ -8,172 +13,480 @@ import Balance from './Balance'
 import Sprite from './Sprite'
 import styles from './EarnReport.module.css'
 
-interface YielgenReportTableProps {
-  lines: string[]
-  maxAmountOfRows?: number
+const SORT_KEYS = {
+  timestamp: 'TIMESTAMP',
+  cjTotalAmountInSats: 'CJ_TOTAL_AMOUNT_IN_SATS',
+  inputCount: 'INPUT_COUNT',
+  inputAmountInSats: 'INPUT_AMOUNT_IN_SATS',
+  feeInSats: 'FEE_IN_SATS',
+  earnedAmountInSats: 'EARNED_AMOUNT_IN_SATS',
 }
 
-const YieldgenReportTable = ({ lines, maxAmountOfRows = 15 }: YielgenReportTableProps) => {
+const TABLE_THEME = {
+  Table: `
+    --data-table-library_grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    font-size: 0.9rem;
+  `,
+  BaseCell: `
+    &:nth-of-type(2) button {
+      display: flex;
+      justify-content: end;
+    }
+    &:nth-of-type(3) button {
+      display: flex;
+      justify-content: center;
+    }
+    &:nth-of-type(4) button {
+      display: flex;
+      justify-content: end;
+    }
+    &:nth-of-type(5) button {
+      display: flex;
+      justify-content: end;
+    }
+    &:nth-of-type(6) button {
+      display: flex;
+      justify-content: end;
+    }
+  `,
+  Cell: `
+    &:nth-of-type(2) {
+      text-align: right;
+    }
+    &:nth-of-type(3) {
+      text-align: center;
```

Comment:
The input count looks a bit lonely centered like this. Does it look better left aligned? 🤔

Reply:
You are right. I narrowed the heading with the suggestion by @MaxHillebrand.. that improved the situation a little bit.

Do you still think it should be left aligned? It looks a bit strange now imho..

![image](https://user-images.githubusercontent.com/3358649/183055446-c5189f30-a8ea-4fc0-a27e-14434b42f66c.png)

as opposed to

![image](https://user-images.githubusercontent.com/3358649/183055545-dd68ea48-ee2a-474c-90c9-542f2b8682c3.png)

Reply:
Let's got with right aligned. Center is weird for a number column in a table and all the others are right aligned as well. See also [here](https://ux.stackexchange.com/a/24073).